github-pr

```diff
@@ -93,17 +117,18 @@ def github_list_prs(**args):
         if args['files']:
             pr_files = pr.get_files()
             args['matching_files'] = [f.filename for f in pr_files]
-        _print_prs([pr], **args)
+        prs = [pr]
     elif args['label']:
         label_list = [repo.get_label(label) for label in args['label']]
         prs = [repo.get_pull(i.number) for i in repo.get_issues(labels=label_list) if i.pull_request]
-        _print_prs(prs, **args)
     elif args['head']:
         prs = _load_prs_by_branch(**args)
-        _print_prs(prs, **args)
+    elif args['merged']:
+        from_date, to_date = _parse_date_fields(['from_date', 'to_date'], **args)
+        prs = _filter_prs_merged_date(repo.get_pulls(state='closed'), from_date, to_date)
```

Comment:
yes, completely agree with you. Definitely is a concern. The API itself does not provide any mechanism to filter by date in any way unfortunately.
We could explore only reading x number of pages from the response, but that introduces another complexity (from which page do we start querying when searching for a specific range of dates?)